### PR TITLE
added record-stats

### PR DESCRIPTION
**Pull request by munishchouhan**
This PR will add record-stats: true for cache to solve the below issue
https://github.com/micronaut-projects/micronaut-cache/issues/306

### CODE REVIEW

**Comment by pditommaso** [comment]
Has this been tested?

**Comment by munishchouhan** [comment]
> Has this been tested?

Tested but still there is no data

**Comment by pditommaso** [comment]
Ouch

**Comment by munishchouhan** [comment]
@pditommaso it's working, i didn't check the graph
Screenshot 2024-06-19 at 15 50 41
Screenshot 2024-06-19 at 15 48 13

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-1.8.2
+1.8.3-A0
```
Comment (pditommaso) [inline VERSION:1]:
```suggestion
1.8.2
```

**Review by pditommaso — APPROVED** [review]

**Pull request merged**